azure-pipelines.dfe.yml

```diff
@@ -124,7 +111,7 @@ jobs:
       - task: UseDotNet@2
```

Comment:
Do we need this task? It looks like we don use it anymore on the above

Reply:
@jamescutts Yes we need it while the vmIMage is 'ubuntu-16.04' because that doesn't include it.